test/impegh_mp4_mux_utils.c

```diff
@@ -978,25 +978,88 @@ static ia_downmixInstruction* selectDownmixInstructions(ia_mpegh3daUniDrcConfig
   return NULL;
 }
 
+static void deriveDrcChannelGroups(
+  const WORD32 drcSetEffect,  /* in */
+  const WORD32 channelCount,  /* in */
+  const WORD8* gainSetIndex,  /* in */
+  UWORD8* nDrcChannelGroups,  /* out */
+  WORD8* uniqueIndex,         /* out (gainSetIndexForChannelGroup) */
+  WORD8* groupForChannel      /* out */
+)
+{
+  WORD32 duckingSequence = -1;
+  WORD32 c, n, g, match, idx;
+  WORD16 uniqueScaling[28];
+
+  for (g = 0; g < 28; g++) {
+    uniqueIndex[g] = -10;
+    uniqueScaling[g] = (WORD16)(-1.0f);
```

Comment:
Local variable set but not referenced anywhere in the function ?

Reply:
Removed.